requirements.txt

```
@@ -16,5 +16,5 @@ selenium==3.14.1
 django-oauth-toolkit==1.5.0
 django-rest-framework-social-oauth2==1.1.0
 spdx-tools==0.8.2
```

```suggestion
spdx-tools==0.8.3
```

NTIA tools version 3.0.2 requires an update to the spdx-tools to 0.8.3